pkg/mtls/testutils/testdata/generate-certs.sh

```diff
@@ -10,12 +10,12 @@ root_ca_exts="
   basicConstraints=CA:TRUE,pathlen:0
 "
 
-openssl req -config <(echo "${root_ca_exts}") -nodes -new -x509 -newkey 2048 -keyout ca-key.pem -out ca.pem -days 1825 -subj "/CN=StackRox Certificate Authority"
+openssl req -config <(echo "${root_ca_exts}") -nodes -new -x509 -newkey 4096 -keyout ca-key.pem -out ca.pem -days 1825 -subj "/CN=StackRox Certificate Authority"
```

Comment:
curious why we updated to 4096. Is this using rsa? Also, this is just testdata, so it doesn't matter much, right? Just curious